solved_exercises/dani/02-tuples.py

```diff
@@ -0,0 +1,43 @@
+import itertools
+'''*********************************************************************************************'''
```

Comment:
It seems you like to use docstrings. Why not use them as they were intended.
Read this PEP: https://www.python.org/dev/peps/pep-0257/

Reply:
I had to expand my browser on the entire monitor to fit this wide code. :))